bigquery/samples/getting_started.py

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env python
+
+# Copyright 2015, Google, Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Sample for making BigQuery queries using the python sdk.
+
+This is a command-line script that queries the public wikipedia-revision
+dataset, and displays the 10 most revised article titles.
+"""
+from pprint import pprint
+
+from apiclient.discovery import build
+from apiclient.errors import HttpError
+
+from oauth2client.client import GoogleCredentials
+
+
+# The id of the project to run queries under.
+PROJECT_ID = 'bigquery-devrel-samples'
```

Comment:
prompt the user and pass this in as an argument to main(), like the [other samples](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/bigquery/samples/sync_query.py#L50).